docs/usage/io.rst

```diff
@@ -0,0 +1,36 @@
+Prediction Using IO Operations
+==============================
+
+This example will show you how to train a model using DFFML python API and use the
+model for prediction by taking input from stdio.
+
+DFFML offers several :ref:`plugin_models`. For this example
+we will be using the Simple Linear Regression model
+( ``slr`` ) which is in the ``dffml`` package.
+
+First we train the model and then create a DataFlow for making predictions
+on user input.
+
+
+.. literalinclude:: /../examples/io/io_usage.py
+
+On running the above code `AcceptUserInput` operation
+waits for input from stdio.
+
+.. code-block:: console
+
+    Enter the value: 21
```

Comment:
Below this code block would be a good place for a dataflow diagram.

https://intel.github.io/dffml/cli.html#diagram

```console
$  dffml service dev export examples.io.io_usage:dataflow | dffml dataflow diagram -config json /dev/stdin
```

You'll paste that into the mermaidjs live editor and then you'll want to save the `.svg`.

But first I'll need to merge https://github.com/intel/dffml/pull/552 because export is currently broken.

Also, you'll need to apply this patch

```diff
diff --git a/examples/io/io_usage.py b/examples/io/io_usage.py
index c3077dc3..d8182a50 100644
--- a/examples/io/io_usage.py
+++ b/examples/io/io_usage.py
@@ -1,7 +1,6 @@
 import asyncio

-from dffml.noasync import train
-from dffml import Features, DefFeature
+from dffml import train, Features, DefFeature
 from dffml.operation.output import GetSingle
 from dffml.df.memory import MemoryOrchestrator
 from dffml.operation.mapping import create_mapping
@@ -16,15 +15,6 @@ slr_model = SLRModel(
     predict=DefFeature("Salary", int, 1),
 )

-# Train the model
-train(
-    slr_model,
-    {"Years": 0, "Salary": 10},
-    {"Years": 1, "Salary": 20},
-    {"Years": 2, "Salary": 30},
-    {"Years": 3, "Salary": 40},
-)
-
 # This Dataflow takes input from stdio using `AcceptUserInput`
 # operation. The string input which corresponds to feature `Years`
 # is converted to `int`/`float` by
@@ -71,8 +61,18 @@ dataflow.seed.append(


 async def main():
+    # Train the model
+    await train(
+        slr_model,
+        {"Years": 0, "Salary": 10},
+        {"Years": 1, "Salary": 20},
+        {"Years": 2, "Salary": 30},
+        {"Years": 3, "Salary": 40},
+    )
+
     async for ctx, results in MemoryOrchestrator.run(dataflow, {"inputs": []}):
         pass


-asyncio.run(main())
+if __name__ == "__main__":
+    asyncio.run(main())
```

Or else when you try to export it will try to run `train`. We should include instructions on generating the diagram in the tutorial as well.